docs/src/modules/spring/pages/spring-boot-integration.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Boot Integration
+
+The Kalix Spring SDK provides a familiar developer experience for Java developers that are used to Spring. Its `Main`
+class is annotated with `@SpringBootApplication` and starts exactly the same way as in a Spring Boot application, namely
+by calling `SpringApplication.run(Main.class, args)`.
+
+For all matters, it's a Spring Boot application that runs and connects with the Kalix Proxy. As such, your application can benefit of most features available to Spring applications. However, some features are not available or are not recommended when building a Kalix application based on the Spring SDK. Those are explained in the sections below.
+
+== Dependency Injection
+
+*Actions*, *Entities* and *Views* are all Kalix components, and they get special treatment.
+
+When your application starts, it will scan the classpath for any class extending one of the Kalix components. Each time such a component is found, it's analysed in order to generate the necessary metadata that Kalix requires to be able to interact with them. The component is then registered to Kalix and made available to the Proxy through Kalix internal protocol.
```

Comment:
Shall we add some further information or a link to explain to the users what the proxy is? Looking at the current docs I don't see a dedicated section for this.
 
I also understand that we would like to avoid the user any knowledge of the proxy if possible. What I don't know is if is it possible at the moment.

Reply:
These ideas are obviously contradictory. I couldn't help thinking I could use an SBR.

Reply:
We don't hide the fact that there is a proxy and the docs talk about it in a few places.

https://docs.kalix.io/reference/glossary.html
https://docs.kalix.io/services/state-model.html
https://docs.kalix.io/services/programming-model.html

Maybe you have only looked at the docs in the jvm project? 

In any case, the goal is not to explain all the internals of the proxy. That would be too much. But it's important that people have a high-level architectural view. A minimal understanding of how Kalix manages state and distributions is unavoidable.

Reply:
I missed it, yeah. Actually, I tried to look up 'proxy' on the search box and as I see now the 'programming model' page appeared quite far in the list. It makes sense, yeah. The users will be better off if they understand the architecture. 

I really like https://docs.kalix.io/services/programming-model.html. Maybe is worth adding a link to that?

Reply:
Thanks, good idea. 

I will the link.